cmd/harborAPI.go

```diff
@@ -561,7 +561,7 @@ func Restart(username string, token string, shipment string, env string) {
 	values := make(map[string]interface{})
 	values["shipment"] = shipment
 	values["env"] = env
-	template, _ := uritemplates.Parse(config.ShipitURI + "/v1/shipment/{shipment}/environment/{env}/envVar/RESTART")
+	template, _ := uritemplates.Parse(config.ShipitURI + "/v1/shipment/{shipment}/environment/{env}/envVar/HC_RESTART")
```

Comment:
Would it make sense to call the envvar HARBOR_x vs HC_x in case we ever decide to also implement a restart command in a GUI or another tool?